frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -0,0 +1,147 @@
+<template lang='pug'>
+  .settings-container
+    section.card
+      h2.is-title-3.c-title Application logs
+
+      .c-header
+        fieldset.c-filters
+          legend.sr-only Visible logs
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='error')
+            i18n Errors
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='warn')
+            i18n Warnings
+          label.checkbox
+            input.input(type='checkbox' name='filter' v-model='filter' value='debug')
+            i18n Debug
+
+        button.is-small.c-download(@click='downloadLogs')
+          i.icon-download.is-prefix.c-icon
+          i18n Download
+          //- i18n.hide-touch Download
+          //- i18n.hide-desktop Share
```

Comment:
Waiting for @mmbotelho decision. If we should display a different text or not based on viewport.

Reply:
@sandrina-p not sure if you saw it but I updated the mockups, yes the copy should change! I also added an icon to the button

https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=9932%3A80581